src/docfx/build/xref/XrefResolver.cs

```diff
@@ -228,6 +248,21 @@ private void ValidateUIDGlobalUnique()
             }
         }
 
+        private void ValidateExternalXref()
+        {
```

Comment:
>xrefGroup.Key [](start = 56, length = 13)

Is it case sensitive?

Reply:
Maybe not, if case sensitive, we should save lower case.